src/bundle-configuration/zookeeper-prod/runtime-config.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
+export ZK_DATA_DIR=/var/lib/zookeeper
```

Comment:
Currently with zookeeper and elasticsearch we have:
- /var/lib/zookeeper
- /var/lib/elasticsearch

Whereas for cassandra we currently have:
- /var/lib/cassandra-v1

We should be consistent across our published bundles.